client/my-sites/plans/current-plan/index.jsx

```diff
@@ -146,7 +147,12 @@ class CurrentPlan extends Component {
 					/>
 					{ isEnabled( 'jetpack/checklist' ) &&
 						isJetpack &&
-						! isAutomatedTransfer && <ChecklistShow /> }
+						! isAutomatedTransfer && (
+							<Fragment>
+								<QueryJetpackPlugins siteIds={ [ selectedSiteId ] } />
```

Comment:
Query 'em early. As long as this query finishes, the guided tour should display immediately.